### ISSUE TITLE
histoid leprosy

### ISSUE THREAD

**Issue by Shreya0802** [obophenotype/human-phenotype-ontology#10863]
**Preferred term label:**
histoid leprosy

**Synonyms**
sarcoidosis, dermatofibroma, cutaneous metastasis, and angiosarcoma

**Definition (free text, please give PubMed ID)**
Histoid leprosy is a rare variant of lepromatous leprosy that is characterized by skin-colored or erythematous papules, plaques, or nodules

**Parent term (use hpo.jax.org/app)**
Appearance: The lesions are well-demarcated, smooth, and globular, and the skin around them appears normal. 
 
Histopathology: The most prominent feature is the presence of many spindle-shaped histiocytes in interlacing bands, whorls, and tight curlicues. 
 
Diagnosis: Fite-Faraco staining can reveal acid-fast and alcohol-fast bacilli.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by mellybelly** [comment]
@Shreya0802 you can make disease requests to mondo here: https://github.com/monarch-initiative/mondo/issues
See also https://mondo.monarchinitiative.org/pages/contributors/ for more information. We are happy to help.
